### PR TITLE
fix: llm generates a response with input containing double quotes

### LINKED ISSUE #1: Exception while handling json in response
Prompt 
Rephrase this as professional email "Hello Ali,
I have received a request from our client to resume the work before the expected timeline. I have checked with they are yet to be assigned to a new project.
Kindly reassign them on erp for the team on a part time hourly basis.
Moreover, I would suggest that we keep looking for new opportunities for both of them as their current assignment with is not full time."

Error 
```
ERROR:root:Error updating message: The request to the Slack API failed. (url: https://www.slack.com/api/chat.update)
The server responded with: {'ok': False, 'error': 'invalid_blocks', 'response_metadata': {'messages': ['[ERROR] failed to match all allowed schemas [json-pointer:/blocks/0/text]', '[ERROR] must be more than 0 characters [json-pointer:/blocks/0/text/text]']}}
Traceback (most recent call last):
  File "/Users/yahya.ali/Documents/office/slack_bot/slack_integration/utils.py", line 64, in update_slack_message
    client.chat_update(
  File "/Users/yahya.ali/Documents/office/slack_bot/venv/lib/python3.9/site-packages/slack_sdk/web/client.py", line 2686, in chat_update
    return self.api_call("chat.update", json=kwargs)
  File "/Users/yahya.ali/Documents/office/slack_bot/venv/lib/python3.9/site-packages/slack_sdk/web/base_client.py", line 155, in api_call
    return self._sync_send(api_url=api_url, req_args=req_args)
  File "/Users/yahya.ali/Documents/office/slack_bot/venv/lib/python3.9/site-packages/slack_sdk/web/base_client.py", line 186, in _sync_send
    return self._urllib_api_call(
  File "/Users/yahya.ali/Documents/office/slack_bot/venv/lib/python3.9/site-packages/slack_sdk/web/base_client.py", line 309, in _urllib_api_call
    return SlackResponse(
  File "/Users/yahya.ali/Documents/office/slack_bot/venv/lib/python3.9/site-packages/slack_sdk/web/slack_response.py", line 199, in validate
    raise e.SlackApiError(message=msg, response=self)
slack_sdk.errors.SlackApiError: The request to the Slack API failed. (url: https://www.slack.com/api/chat.update)
The server responded with: {'ok': False, 'error': 'invalid_blocks', 'response_metadata': {'messages': ['[ERROR] failed to match all allowed schemas [json-pointer:/blocks/0/text]', '[ERROR] must be more than 0 characters [json-pointer:/blocks/0/text/text]']}}

```

### PR DESCRIPTION
**Pull request by minahilfaisal**
**Issue Resolved:**
Closes #1.

**Description:**
This PR fixes the exception received while handling json in response. The exception was occurring due to the llm not being able to generate a response when the input text contained double quotes like in the following prompt:

> Rephrase this as professional email "Hello Ali,
> I have received a request from our client to resume the work before the expected timeline. I have checked with they are yet to be assigned to a new project.
> Kindly reassign them on erp for the team on a part time hourly basis.
> Moreover, I would suggest that we keep looking for new opportunities for both of them as their current assignment with is not full time."

The function `get_ai_response()` returned `null` which raised the exception when trying to send the response to slack.

**Fix:**

The modifications implemented include:

1. Updated the `get_llm_answer()` function to replace double quotes with angled brackets ("<" at the beginning and ">" at the end) so that the cleaned text sent to the llm generates response.
2. Addition of an extra measure to check if the language model (LLM) generated a response; if not, an error message is sent to the user.

### CODE REVIEW

**Comment by Yahyaali1** [comment]
@minahilfaisal looks good. Lets add a unit test case to make sure this works as expected ?

File: tests/test_llm.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        and doube quotes."""
+        question = 'How are you? <http://example.com/image.jpg|image>, hehe :smiley:, "I am good".'
+        expected_clean_text = 'How are you? , hehe , <I am good>.'
+        with patch('llm_integration.llm.llm_prompt') as mock_llm_prompt:
+            mock_llm_prompt.return_value = "Response"
+            get_llm_answer(question)
+            mock_llm_prompt.assert_called_once_with(question=expected_clean_text)
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (Yahyaali1) [inline tests/test_llm.py:52]: Do we need this ? 
We can run tests directly.

**Review by Yahyaali1 — APPROVED** [review]

File: tests/test_llm.py (hunk 1 above)
Comment (minahilfaisal) [reply tests/test_llm.py]: Updated.

**Review by Yahyaali1 — APPROVED** [review]
